activities/tool-management/nextcloud.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# Nextcloud
+
+Nextcloud is a suite of client-server software for creating and using file hosting services. Nextcloud is free and open-source. We are currenty using Nextcloud for file sharing with our community and the general public.
+
+## Using Nextcloud
+
+Staff are issued a Nextcloud account when they join. Nextcloud publishes [extensive support documentation](https://docs.nextcloud.com/).
+
+## Nextcloud management
+
+Nextcloud (user) administration is managed by the Operations Coordinator. The Operations Coordinator as well as the Chief Executive have full adminstrative privileges. The Operations Coordinator is responsible for user and folder management, customization, making sure Nextcloud stays up-to-date and ensures proper on-boarding and off-boarding policies are in place and enforced.
+
+## Privacy and GDPR
+
+Nextcloud has [an extensive GDPR complaince kit.](https://nextcloud.com/gdpr/)
+
+> "Nextcloud offers everything you need for GDPR compliance. Compliance is a multi-step process and data can be in many locations. Our documentation helps administrators check if they have covered their bases and guides them through delivering on data access, modification and deletion requests. We offer both a high level overview as well as hands-on, concrete documentation for administrators."
```

Comment:
Re Nextcloud GDPR, it would be helpful to say which data is collected (for example, data from people who download files or only staff?), how the data's being hosted, where (outside the EU?), and who has access to it.

Reply:
It looks like I need to request the GDPR kit before we can get more of these details, which I'll do now: https://nextcloud.com/gdpr/

Reply:
Hey, it looks like this got merged without the GDPR section being updated to explain any of the stuff I mentioned above. Were you able to get that information?